File: hw/segarra/u2/HW18ZooInPythonAndJavaScript/ec.edu.zooinjavascript.model/Tiger.js
import Feline from "./Feline.js"

export default class Tiger extends Feline{
    constructor(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate){
        super(id, description, bornOnDate, gender, cage, numberOfBones, vertebrate)
    }

    feed(){
        console.log('Feeding the tiger with meat')
    }
}

File: ws/segarra/u2/WS21Inheretance/WS21Inheritance/src/main/java/ec/edu/espe/WS21Inheritance/model/Tiger.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.WS21Inheritance.model;

import java.util.Date;

/**
 *
 * @author Eduardo Segarra, TheJavaBandits, DCCO-ESPE
 */
public class Tiger extends Feline{

    public Tiger(int id, String description, Date bornOnDate, char gender, Cage cage, int numberOfBones, boolean vetebrate) {
        super(id, description, bornOnDate, gender, cage, numberOfBones, vetebrate);
    }

    @Override
    public void feed(int food) {
        System.out.println("feeding the lion with meat");
    }
    
}
